python/MooseDocs/test/extensions/test_versioner.py

```diff
@@ -59,8 +59,14 @@ def testCondaVersionRender(self):
     def testCodeRender(self):
         """Test inline replacement within code blocks"""
         for package in versioner.TRACKING_LIBRARIES:
-            # Application doesn't have a conda version
-            has_conda_version = package != 'app'
+            # Application, doesn't have a conda version
+            # Several support libraries also have no conda version
+            has_conda_version = package not in ['app',
+                                                'libmesh-vtk',
+                                                'peacock',
+                                                'pprof',
+                                                'pyhit',
+                                                'seacas']
```

Comment:
@loganharbour I am not familiar with these things not having a 'conda version'. Especially the new tracking additions. But without these, I do get errors. I want to make sure I am doing this right.

error:
```pre
Traceback (most recent call last):
  File "/Users/milljm/projects/moose/python/MooseDocs/test/extensions/test_versioner.py", line 80, in testCodeRender
    conda_version = self.version_meta[package]['conda']['version_and_build']
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
KeyError: 'version_and_build'
```
I suppose I need to see how we're building this dictionary...

Reply:
I separated what versioner.py can keep track of, over what unittest is verifying in versioner_hashes.yaml. We don't want to keep track of every single artifact in versioner_hashes.yaml...